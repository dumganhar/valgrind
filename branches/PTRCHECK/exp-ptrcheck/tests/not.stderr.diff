5,7c5
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
---
>  Address 0x........ is not derived from any known block
12,14c10
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
---
>  Address 0x........ is not derived from any known block
16,21d11
< 
< Invalid read of size 4
<    at 0x........: main (not.c:16)
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
23,28d12
< 
< Invalid read of size 4
<    at 0x........: main (not.c:16)
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
33,35c17
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
---
>  Address 0x........ is not derived from any known block
40,42c22
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
---
>  Address 0x........ is not derived from any known block
47,49c27
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
---
>  Address 0x........ is not derived from any known block
54,56c32
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
---
>  Address 0x........ is not derived from any known block
61,63c37
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
---
>  Address 0x........ is not derived from any known block
68,70c42
<    by 0x........: __libc_start_main (...libc...)
<    by 0x........: ...
< Address 0x........ is not derived from any known block
---
>  Address 0x........ is not derived from any known block
72c44
< ERROR SUMMARY: 10 errors from 10 contexts (suppressed: 0 from 0)
---
> ERROR SUMMARY: 8 errors from 8 contexts (suppressed: 0 from 0)
