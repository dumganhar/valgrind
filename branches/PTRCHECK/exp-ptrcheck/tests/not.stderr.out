
about to do 14 [0]
Invalid read of size 4
   at 0x........: main (not.c:14)
 Address 0x........ is not derived from any known block
about to do 14 [-1]

Invalid read of size 4
   at 0x........: main (not.c:14)
 Address 0x........ is not derived from any known block
about to do 16 [0]
about to do 16 [-1]
about to do 18 [0]

Invalid read of size 4
   at 0x........: main (not.c:18)
 Address 0x........ is not derived from any known block
about to do 18 [-1]

Invalid read of size 4
   at 0x........: main (not.c:18)
 Address 0x........ is not derived from any known block
about to do 20 [0]

Invalid read of size 4
   at 0x........: main (not.c:20)
 Address 0x........ is not derived from any known block
about to do 20 [-1]

Invalid read of size 4
   at 0x........: main (not.c:20)
 Address 0x........ is not derived from any known block
about to do 22 [0]

Invalid read of size 4
   at 0x........: main (not.c:22)
 Address 0x........ is not derived from any known block
about to do 22 [-1]

Invalid read of size 4
   at 0x........: main (not.c:22)
 Address 0x........ is not derived from any known block

ERROR SUMMARY: 8 errors from 8 contexts (suppressed: 0 from 0)
